sphinx_gallery/tests/test_full.py

```diff
@@ -37,8 +37,8 @@ def ignore(src, names):
     with docutils_namespace():
         app = Sphinx(src_dir, conf_dir, out_dir, toctrees_dir,
                      buildername='html')
-
-    app.build(False, [])
+        # need to build with this mode for automodule and backrefs to work
```

Comment:
Not sure what this comment is about ... what do you mean by "this mode"? Is that related to moving `app.build` inside the `with docutils_namespace()`

Reply:
yes exactly, if the `app.build` is not in the context manager the links are not actually made